packages/node/src/utils/types.ts

```diff
@@ -1,3 +1,5 @@
+import ts = require('typescript');
```

Comment:
We should be able to use `import type { } from 'typescript';` so this file doesn't bring in `typescript` (which is heavy).

Reply:
Changes made. Thanks Jason